robot/robot.go

```diff
@@ -89,6 +89,8 @@ type Robot interface {
 
 	// StopAll cancels all current and outstanding operations for the robot and stops all actuators and movement
 	StopAll(ctx context.Context, extra map[resource.Name]map[string]interface{}) error
+
+	RestartModule(ctx context.Context, req RestartModuleRequest) error
```

Comment:
```suggestion

        // RestartModule reloads a module as if its config changed
	RestartModule(ctx context.Context, req RestartModuleRequest) error
```

This func is exposed to all users of the go sdk I think. We probably want to add a comment

Reply:
Not sure -- I might be wrong -- can ignore if I am wrong